src/main/java/org/jenkinsci/plugins/docker/workflow/client/DockerClient.java

```diff
@@ -178,8 +199,9 @@ private Date getCreatedDate(@Nonnull EnvVars launchEnv, @Nonnull String objectId
      */
     public @CheckForNull VersionNumber version() throws IOException, InterruptedException {
         LaunchResult result = launch(new EnvVars(), true, "-v");
+        final String stdout = result.getOut();
         if (result.getStatus() == 0) {
-            return parseVersionNumber(result.getOut());
+            return parseVersionNumber(stdout);
```

Comment:
Seems to be an unnecessary diff hunk, perhaps left over from a previous version. Harmless though.